cmd/identities/list.go

```diff
@@ -0,0 +1,53 @@
+package identities
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/ory/kratos/internal/clihelpers"
+
+	"github.com/spf13/cobra"
+
+	"github.com/ory/kratos/cmd/cliclient"
+	"github.com/ory/kratos/internal/httpclient/client/admin"
+	"github.com/ory/x/cmdx"
+)
+
+var listCmd = &cobra.Command{
+	Use: "list [<page> <per-page>]",
+	Args: func(cmd *cobra.Command, args []string) error {
+		// zero or exactly two args
+		if len(args) != 0 && len(args) != 2 {
```

Comment:
cobra.Command has something called `Args` where you can use `cobra.ExactArgs` :)

Reply:
Sure but it can not check for 0 *or* 2 args